bin/check-rds-events.rb

```diff
@@ -52,6 +59,11 @@ class CheckRDSEvents < Sensu::Plugin::Check::CLI
          description: 'AWS Region (defaults to us-east-1).',
          default:     'us-east-1'
 
+  option :db_instance_id,
+         short:       '-i N',
+         long:        '--db-instance-id NAME',
+         description: 'DB instance identifier'
```

Comment:
Is there a particular part you would like me to expand upon? Should I increase the the content of the description? The option here is actually copy/paste from the only other rds check `check-rds.rb`. I figured consistency between the two would reduce confusion .